security-privacy-questionaire.md

```diff
@@ -0,0 +1,97 @@
+# WebCodecs - Security and Privacy Questionnaire
+
+This document answers the [W3C Security and Privacy
+Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/) for the
+WebCodecs specification.
+
+Last Update: 2019-10-18
+
+**2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?**
+
+This feature may expose additional information about what media a user agent is
```

Comment:
Could also clarify that this is mostly a concern for hardware encoders/decoders, as I expect software codecs will be coupled with the browser and version.